packages/nextjs/src/server/index.ts

```diff
@@ -146,21 +140,16 @@ function addServerIntegrations(options: NextjsOptions): void {
 
 // TODO (v8): Remove this
 /**
- * @deprecated Use the constant `IS_BUILD` instead.
+ * @deprecated
```

Comment:
I don't know why or how this API got to the point that it was exported but IMO we shouldn't export this. It is completely unrelated to what Sentry is doing so I wouldn't tell users to use either `isBuild` or `IS_BUILD`.

I will move this change into a separate PR so that it appears in the changelog.